### PR TITLE
Update default iOS Simulator

### PR DESCRIPTION
**Pull request by smohamedjavid**
### Summary

This PR updates the iOS default simulator to `iPhone 11 Pro` as its screen dimensions match with our design.

Also, updates the document to include steps to add `iPhone 11 Pro` simulator from your Xcode.

status: ready

### CODE REVIEW

**Review by briansztamfater — APPROVED** [review]

**Comment by J-Son89** [comment]
hmm, I would actually think the default should be iPhone 11 Pro as that is what we are matching the figma designs to. Maybe we should discuss this in mobile channel first? 🤔

File: Makefile [hunk 1]
```diff
@@ -268,7 +268,7 @@ run-android: export TARGET := android
 run-android: ##@run Build Android APK and start it on the device
 	npx react-native run-android --appIdSuffix debug
 
-SIMULATOR=iPhone 13
+SIMULATOR=iPhone 14
```
Comment (flexsurfer) [inline Makefile]: it should be `iPhone 14 Pro` because of the screen size, its identical to figma designs

**Review by flexsurfer — CHANGES_REQUESTED** [review]

**Comment by briansztamfater** [comment]
> hmm, I would actually think the default should be iPhone 11 Pro as that is what we are matching the figma designs to. Maybe we should discuss this in mobile channel first? 🤔

I think 11 Pro is too old, as new Xcode versions are released, old simulators are removed and not installed by default, so having 14 or 14 Pro as default makes more sense to me.

File: Makefile (hunk 1 above)
Comment (briansztamfater) [reply Makefile]: Yeah good point, I wondered why we didn't have 13 Pro as default before, I think 14 Pro is the best option now

**Comment by J-Son89** [comment]
> > hmm, I would actually think the default should be iPhone 11 Pro as that is what we are matching the figma designs to. Maybe we should discuss this in mobile channel first? 🤔
> 
> I think 11 Pro is too old, as new Xcode versions are released, old simulators are removed and not installed by default, so having 14 or 14 Pro as default makes more sense to me.

whatever is the same size as the Figma size is fine. @flexsurfer says 14 pro works with Figma so let's go with that. 👍

**Review by J-Son89 — APPROVED** [review]

**Comment by smohamedjavid** [comment]
Our designs use a `375x812` layout/dimension which is `iPhone X/XS/11 Pro`.

If we use `iPhone 14 Pro`, the design is supposed to be a `393x852` layout/dimension referring to the periodic table of iPhone Screen Size from [Figma](https://www.figma.com/community/file/905537113759977194/The-Periodic-Table-of-iPhone-Screen-Sizes)

Screenshot 2023-06-15 at 5 10 43 PM

<hr />

If we have to use the old simulator (`iPhone 11 Pro`), it requires extra steps to add in Xcode else the build will fail.

1. Open Xcode
2. Menu > Window > Devices and Simulators
3. Tap `+` button on bottom left
4. Select Device Type as `iPhone 11 Pro` and tap on `Create`

**Comment by flexsurfer** [comment]
yes sorry, i confused 11 and 14, i meant **11** Pro

**Comment by J-Son89** [comment]
> Our designs use a `375x812` layout/dimension which is `iPhone X/XS/11 Pro`.
> 
> If we use `iPhone 14 Pro`, the design is supposed to be a `393x852` layout/dimension referring to the periodic table of iPhone Screen Size from [Figma](https://www.figma.com/community/file/905537113759977194/The-Periodic-Table-of-iPhone-Screen-Sizes)
> 
> Screenshot 2023-06-15 at 5 10 43 PM
> If we have to use the old simulator (`iPhone 11 Pro`), it requires extra steps to add in Xcode else the build will fail.
> 
> 1. Open Xcode
> 2. Menu > Window > Devices and Simulators
> 3. Tap `+` button on bottom left
> 4. Select Device Type as `iPhone 11 Pro` and tap on `Create`

yep that's fine, actually I was not aware of this list of screen sizes at all and seems that we can use any of these devices :) 
It would be great to have this in a doc and to explain how to match your device with Figma size etc.

File: Makefile (hunk 1 above)
Comment (flexsurfer) [reply Makefile]: i meant `iPhone 11 Pro` sorry

**Review by flexsurfer — APPROVED** [review]
thanks

**Comment by smohamedjavid** [comment]
The tests are failing due to the `slide-button` test suite.

**Review by siddarthkay — APPROVED** [review]

**Pull request merged**